campair/campair/ViewModel/EditorViewModel.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ViewModelPlaceHolder.swift
+//  campair
+//
+//  Created by Shin Jae Ung on 2022/06/10.
+//
+
+import Foundation
```

Comment:
```suggestion
import Combine
import Foundation
```

사용되는 모듈을 한 눈에 보여주면 읽는쪽에서 어느 프레임워크를 사용해서 구현하는구나 라고 한눈에 알아보기 쉬워집니다
ObservableObject가 사용되는 것 같으니 콤바인도 추가합시다요 ~~~~

Reply:
ViewModel should import `SwiftUI` instead of `Foundation`.

Reply:
> ViewModel should import `SwiftUI` instead of `Foundation`.

@sustainable-git Young maybe said we should clarify framework what we used for. So I followed her comment.